lib/models/showtimes.dart

```diff
@@ -5,11 +5,11 @@ import 'package:cinetime/utils/_utils.dart';
 import '_models.dart';
 
 class MoviesShowTimes {
-  final bool? fromCache;
-  final List<MovieShowTimes>? moviesShowTimes;
-  final DateTime? fetchedAt;
-
   const MoviesShowTimes({this.fetchedAt, this.fromCache, this.moviesShowTimes});
+
+  final List<MovieShowTimes>? moviesShowTimes;
+  final bool? fromCache;        // TODO remove not used ?
```

Comment:
Android Studio should be able to tell you if unused?

Reply:
Sure, and it's not ;) 
Question it : should I remove it, or implement it ?

Reply:
Is it not gonna use the cache automatically?

Reply:
This is an old field, used to indicate whether the data displayed is from cache or from server.
But it hasn't been implemented in the UI